### ISSUE TITLE
Use object-fit css property in Background component

### ISSUE THREAD

**Issue by a-axton** [yankaindustries/mc-components#611]
Instead of doing math to calculate positioning an image inside of Background we should use the object-fit property. I've been having issues with passing an img component into TileImage because the ref wasn't being properly passed. This would simplify that component making it more adaptable. We're also dropping support for IE so we should be fine to use this now. 

Here's the code that worked for me

```
&__lazy-tile {
    display: flex;
    flex-direction: column;
    width: 100%;
    height: 100%;

    img {
      object-fit: contain;
      width: 100%;
    }
  }
```

**Comment by stephenp** [comment]
Yeah, if we're able to drop IE11 support, this could work, but until then, this solution won't work for our implementation: https://caniuse.com/#feat=object-fit

**Comment by a-axton** [comment]
We pretty much dropped support as soon as we added css variables, have you seen the site in IE 11? It's not just css variables, there seems to be various things not supported. I checked on Friday and it looks wrecked 
![IMG_20191101_163656](https://user-images.githubusercontent.com/4015654/68148914-c78d2400-fef1-11e9-96c0-5edf0b1b3cd3.jpg)

**Comment by stephenp** [comment]
We didn't drop support with css variables - the variables were added, but the old scss variables were kept as well.  But yeah, the site does look wrecked.

**Comment by stephenp** [comment]
In any case, apparently we _have_ dropped ie 11 support, so this looks good!  This issue's pr should both implement the object-fit property as well as remove any js sizing.
